git/git_branches_intro.md

```diff
@@ -1,22 +1,28 @@
 # Git Branches Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+This lesson is focused on one of the key features of Git: branches. Branches allow developers to create a fresh copy of their repositories and experiment. Branches can be useful for testing new ideas, compartmentalizing projects, and allowing multiple developers to contribute to the same project simultaneously.
 
 ## Learning outcomes
 *Look through these now and then use them to test yourself after doing the assignment*
 
-what the student is expected to know or be able to do by the end of this lesson
-
-* Learning outcome 1
-* Learning outcome 2
-* Learning outcome 3
+* Be able to answer the question, "What is a branch?"
+* Have a cursory knowledge of Git workflows
+* Know how to create a new branch
+* Know how to merge a branch
+* Know how to delete a branch
 
 ## Assignment
-The list of resources the user will go through to learn about the topic of this lesson. Have no more than 5 resources, ideally no more than three.
+1. Read the beginner-friendly [Using Branches](https://www.atlassian.com/git/tutorials/using-branches) tutorial from [Atlassian](https://www.atlassian.com/)
+2. Watch [GIT: Working with Branches](https://www.youtube.com/watch?v=JTE2Fn_sCZs) from [Codemy School](https://www.codemy.net/)
+3. [Optional] Read about different [Git Workflows](https://www.atlassian.com/git/tutorials/comparing-workflows) to understand the application of Git branches.
 
 ## Exercises
-A group of exercises (If Applicable) for the student to complete in relation to the topic taught in the lesson.
+* Complete exercises 1.18 and 1.19 on [Try Git](https://try.github.io/levels/1/challenges/18).
+* Complete the *Introduction Sequence* on [learngitbranching.js.org](http://learngitbranching.js.org/)
```

Comment:
Sounds good. The Git basics lesson already shares Try Git, so it will still be a resource to students.